tests/selectedtests/app/controllers/test_health_controller.py

```diff
@@ -7,4 +7,5 @@ def test_health_endpoint(app_client: testing.FlaskClient):
     """
     response = app_client.get("/health")
     assert response.status_code == 200
-    assert "online" and "true" in response.get_data(as_text=True)
+    res_data = response.get_data(as_text=True)
+    assert "online" in res_data and "true" in res_data
```

Comment:
I think it might be good to check the json structure as well. This will pass if the API returns "onlinetrue".

```
def test_health_endpoint(app_client: testing.FlaskClient):
    """
    Test /health endpoint
    """
    response = app_client.get("/health")
    assert response.status_code == 200
    response_data = response.get_json()
    assert "online" in response_data and response_data['online']
```